packages/component/src/createLoadable.js

```diff
@@ -90,7 +90,17 @@ function createLoadable({ resolve = identity, render, onLoad }) {
       componentDidMount() {
         this.mounted = true
 
-        if (this.state.loading) {
+        const cachedPromise = this.getCache()
+
+        if (cachedPromise && cachedPromise.status === STATUS_REJECTED) {
+          this.setCache()
+          this.setState({
+            error: undefined,
+            loading: true,
+          })
+
+          if (!options.suspense) this.loadAsync()
```

Comment:
You can load it async with suspense as well, why not?

Reply:
Because you need to throw the loadAsync call on the render method so that Suspense catches it, doing that on DidMount makes you add a lot more if conditions at the render method to support more corner cases....sometimes the Best Solution to a problem is to avoid that problem :/

Reply:
@theKashey 
Besides adding a fallback for Proxy, is there anything else preventing the acceptance of this PR?